pkg/ebpf/c/tracee.bpf.c

```diff
@@ -5296,17 +5296,30 @@ int BPF_KPROBE(trace_security_bpf)
     return events_perf_submit(&data, SECURITY_BPF, 0);
 }
 
-SEC("kprobe/arm_kprobe")
-int BPF_KPROBE(trace_arm_kprobe)
+SEC("kprobe/enable_kprobe")
```

Comment:
@roikol FYI. I'm not sure how this event is being consumed from Research, but I'm addressing fixes for arm64 (Issue: #1991) and thought this could be a fit.